dask_expr/tests/test_collection.py

```diff
@@ -232,6 +232,17 @@ def test_columns_traverse_filters(pdf, df):
     assert str(result) == str(expected)
 
 
+@pytest.mark.parametrize("projection", ["zz", ["zz"], ["zz", "x"], "zz"])
+@pytest.mark.parametrize("subset", ["x", ["x"]])
+def test_drop_duplicates_subset_optimizing(pdf, subset, projection):
```

Comment:
One more note, there's been an effort in the project to keep testing times very fast (we were sub-two-seconds for a long time, that went away when we added shuffling).  Seeing many-parameter tests for simple functionality has me mildly concerned, but maybe it's appropriate at this point to start caring more about coverage than about speed.

cc @rjzamora in case he wants to weigh in

Mostly I want to convey "tests are currently fast, I'd love it if they stayed that way"

Reply:
Yeah this is mostly a coverage question, will defer to both of you there. One note though: none of these are redundant coverage wise

Reply:
Probably the right answer is to prefer coverage at this point (even if it makes me a little sad)

Reply:
Coverage should probably be the priority. However, we should certainly try to minimize run time to whatever extent we can. What is the "delta" on the new test coverage here?

Reply:
The whole test runs in a bit under 30ms locally, so should be good? ``test_setitem`` takes over 100ms as a comparison.